charms/worker/k8s/src/token_distributor.py

```diff
@@ -479,19 +515,15 @@ def revoke_tokens(
             ",".join(sorted(u.name for u in remaining)),
         )
 
-        tokenizer = self.token_strategies.get(token_strategy)
-        assert tokenizer, f"Invalid token_strategy: {token_strategy}"  # nosec
-
```

Comment:
pylint was fussing about too many local variables.  This is a bit of code golf to slim down the variables